doc/compatibility_matrix.rst

```diff
@@ -10,39 +10,53 @@ The following table demonstrates the compatibility between the nRF Connect SDK a
    :widths: auto
    :header-rows: 1
 
-   * - nRF Connect SDK version
+   * - Add-on version
+     - nRF Connect SDK version
      - Amazon specification (Sidewalk MCU SDK version)
      - Supported platforms
      - The Amazon Sidewalk Protocol Implementation Conformance Statement (PICS)
      - Amazon qualification status
-   * - v2.9.0
+   * - v0.1.0
+     - v3.0.0
+     - #######TBA
+     - * `nRF52840 DK`_
+       * `nRF5340 DK`_
+       * `nRF54L15 DK`_
+     - #######TBA
+     - #######TBA
+   * - --
+     - v2.9.0
      - v1.17.0
      - * `nRF52840 DK`_
        * `nRF5340 DK`_
        * `nRF54L15 DK`_
      - * `PICS v2.9.0 - BLE only`_
        * `PICS v2.9.0`_
      - Planned
-   * - v2.8.0
+   * - --
+     - v2.8.0
      - v1.17.0
      - * `nRF52840 DK`_
        * `nRF5340 DK`_
        * `nRF54L15 DK`_
      - --
      - Not planned
-   * - v2.7.0
+   * - --
+     - v2.7.0
      - v1.16.2
      - * `nRF52840 DK`_
        * `nRF5340 DK`_
      - ---
      - Qualified/approved
-   * - v2.6.0
+   * - --
+     - v2.6.0
      - v1.16.2
      - * `nRF52840 DK`_
        * `nRF5340 DK`_
      - ---
      - Qualified/approved
-   * - v2.5.0
+   * - --
+     - v2.5.0
      - v1.14
      - * `nRF52840 DK`_
        * `nRF5340 DK`_

```

Comment:
So after all we want to keep the old releases ? 

I thought we want to start only with add-on releases